src/ripple/test/jtx/Account.h

```diff
@@ -115,6 +109,20 @@ class Account
     /** Returns an IOU for the specified gateway currency. */
     IOU
     operator[](std::string const& s) const;
+
+private:
+    Account (std::string name,
+        std::pair<PublicKey, SecretKey> const& keys);
+
+    static std::unordered_map<
+        std::pair<std::string, KeyType>,
+            Account, beast::uhash<>> cache_;
```

Comment:
`cashe_` could be a static inside the constructor (or inside the `fromCache` helper function if you take that change).